app/views/About.js

```diff
@@ -84,12 +85,12 @@ class AboutScreen extends Component {
           <View style={styles.spacer} />
 
           <SvgXml style={styles.aboutSectionIconLock} xml={lock} />
-          <Text style={styles.aboutSectionTitles}>
+          <DynamicText style={styles.aboutSectionTitles}>
```

Comment:
Please converge on `Typography`, all text will eventually use `<Typography>`, we are gradually converging on fixed headline/body styles from [Figma](https://www.figma.com/file/n8fjoYkgJybfYgrOvLRMBf/Safe-Paths---Draft-designs?node-id=1358%3A11099)

Reply:
Done, and need to choose the "bodyX|headlineX" (Put I prefer making this with new branch after merging this for styling task only)